### ISSUE TITLE
Yarn (1.22.22) throws warning "Workspaces can only be enabled in private projects"

### ISSUE THREAD

**Issue by Roozenboom** [playwright-community/eslint-plugin-playwright#360]
After installing `eslint-plugin-playwright` in our workspace we see a warning every time we do a yarn install.
```
warning Workspaces can only be enabled in private projects.
```
This warning is caused because in the published package.json this plugin has:
```
  "workspaces": [
    "examples"
  ],
```
I have seen this same issue with many other packages and it depends on the package manager (yarn classic / v1) you use if you see this warning or not. The easiest solution is to publish a clean package.json that does not contains any development data like workspaces as this info is not required when you install the package.
